server/e2e/gql_storytelling_test.go

```
@@ -7,12 +7,402 @@
+
+	updatePropertyValue(e, blockPropID, "default", "", "text", "test value", "STRING")
+
+	p := map[string]any{"left": 0, "right": 1, "top": 2, "bottom": 3}
+	updatePropertyValue(e, blockPropID, "panel", "", "padding", p, "SPACING")
+
+	publishStory(e, storyID1, "test-alias")
+
+	rc, err := g.File.ReadStoryFile(context.Background(), "test-alias")
+	assert.NoError(t, err)
+
+	expected := fmt.Sprintf(`
```

golangci-lint (1.62.2)

printf: fmt.Sprintf call has arguments but no formatting directives

(govet)